apple/RNCWebView.m

```diff
@@ -456,7 +456,7 @@ - (void)didMoveToWindow
   if (self.menuItems != nil) {
     UILongPressGestureRecognizer *longPress = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(startLongPress:)];
     longPress.delegate = self;
-
+    
```

Comment:
Not a big deal, but weird to be that an autoformatting tool would add unnecessary whitespace like this....